src/sentry/api/endpoints/group_ai_autofix.py

```diff
@@ -158,8 +142,10 @@ def post(self, request: Request, group: Group) -> Response:
             ],
         }
 
-        if not request.user.is_authenticated:
-            raise PermissionDenied(detail="You must be authenticated to perform this action.")
+        if not features.has("projects:ai-autofix", group.project):
+            return self._respond_with_error(
+                group, metadata, "AI Autofix is not enabled for this project.", 403
```

Comment:
Are we consistently calling it "AI Autofix"

Reply:
yes, on the product itself it has been